src/picknik_dual_ur_arm_config/waypoints/ur_waypoints.yaml

```diff
@@ -0,0 +1,79 @@
+- description: 'Both arms are in a down position'
+  favorite: true
+  joint_state:
+    effort: []
+    header:
+      frame_id: world
+      stamp:
+        nanosec: 0
+        sec: 0
+    name:
+    - left_shoulder_pan_joint
+    - left_shoulder_lift_joint
+    - left_elbow_joint
+    - left_wrist_1_joint
+    - left_wrist_2_joint
+    - left_wrist_3_joint
+    - right_shoulder_pan_joint
+    - right_shoulder_lift_joint
+    - right_elbow_joint
+    - right_wrist_1_joint
+    - right_wrist_2_joint
+    - right_wrist_3_joint
```

Comment:
This is something I couldn't figure out with the waypoint serializer, but for some reason the resulting lists saved violate our formatter's indentation check.

Unfortunately, these are not resolvable automatically so you'll need to go in and fix by hand... or find how we can add this indent in the source code itself (which I couldn't...)

Reply:
Fixed it and created an issue